seatunnel-examples/seatunnel-flink-connector-v2-example/pom.xml

```diff
@@ -18,8 +18,8 @@
 
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
revert this change

Do not submit example module

Reply:
gitignore file contains `seatunnel-examples ` , i am curious about why doesn't it take effect

Reply:
> gitignore file contains `seatunnel-examples ` , i am curious about why doesn't it take effect

The `.gitignore` only ensure the newly added files are not checked, but the existing files are still checked for changes.